distros/sandbox/gitea/statefulset-gitea.yaml

```diff
@@ -46,7 +46,10 @@ spec:
               mountPath: /tmp
             - name: data
               mountPath: /data
-          securityContext: {}
+          securityContext:
+            capabilities:
+              add:
+                - CAP_CHOWN
```

Comment:
FYI: The init-directories containers needs the `CAP_CHOWN` capability, otherwise, it will with the below error
```
ubuntu@ubuntu:~$ kubectl logs gitea-0 -n gitea -c init-directories
+ chown 1000:1000 /data
chown: /data: Operation not permitted
```